api/net/Areas/Editor/Controllers/ContentController.cs

```diff
@@ -377,7 +377,11 @@ public async Task<IActionResult> DeleteAsync(ContentModel model)
     [SwaggerOperation(Tags = new[] { "Content" })]
     public async Task<IActionResult> PublishAsync(ContentModel model)
     {
-        if (model.Status != ContentStatus.Published) model.Status = ContentStatus.Publish;
+        var result = _contentService.FindById(model.Id);
```

Comment:
Remove your change here.  Please review code usage before making changes.
1) Currently there is no frontend app or backend service using this endpoint.  I have considered removing it at some point, but it may find use in a future scenario.
2) The status should be changed to `Publish` so that the Indexing Service updates it to `Published` when it has actually been published to Elasticsearch.  The status of content informs us where it is.  While content can already have been published, when an editor publishes it again, we change the state in the DB to publish to inform us of the lifecycle.
3) Your logic would never let us update a published content item.

Reply:
Removed, maybe we should comment out the endpoint here as we are not using it for the moment. Thanks